### DISCUSSION TITLE
When dropdown opens the list does not appear just below using modal

### DISCUSSION THREAD

**Discussion by gbspecapedra** [MobileReality/react-native-select-pro#125] [Q&A] [answered]
Hello folks, I'm using the `Select` component inside a react native Modal. I have used `SelectModalProvider` as follows in the documentation, however, the modal has the `presentationStyle="pageSheet"`, so when the dropdown opens the list does not appear just below.

It works correctly without the attribute `presentationStyle="pageSheet"` in the modal, but now this attribute is a requirement. Does anyone have any idea how to fix it?

![Xnip2022-11-09_11-13-38](https://user-images.githubusercontent.com/97048648/200852974-ab28c8c5-135e-4750-90d9-af4a9d773684.jpg)

**Comment by kkaplinski** [comment]
Hi @wdgisele, Thanks for the reporting this behaviour. We will investigate it. 👍

**Comment by irekrog** [comment, marked as answer]
Hi @wdgisele 

You can check out the last version of the lib (1.13.1, there is a fix for pageSheet mode) + to fix a horizontal position you should set [margin: 0](https://github.com/react-native-modal/react-native-modal#the-modal-is-not-covering-the-entire-screen)
